Formula/gerbil-scheme.rb

```diff
@@ -47,6 +47,15 @@ def install
     end
   end
 
+  def caveats; <<~EOS
```

Comment:
I assume that these are not Homebrew specific caveats, therefore we do not want to add them.

cc @MikeMcQuaid for opinions

Reply:
Agreed 👍

Reply:
These are Homebrew specific caveats.
We are not allowed to install non-binaries in lib.
This creates problems for a dynamic language where bin/foo expects lib/foo.ss to exist.
See the previous history. This caveat is already in the upstream gerbil documentation.
Due to the fact we must put non-binaries into libexec/lib instead of lib/, that means the binaries have to go into libexec/bin. A symlink won't work without these variables set.

Reply:
> This caveat is already in the upstream gerbil documentation.

That means it doesn't need duplicated here.